packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1171,6 +1179,11 @@ export interface CameraSitePermissionsProps extends CommonSitePermissionsProps {
 // @beta
 export type CameraSitePermissionsStrings = SitePermissionsStrings;
 
+// @public
```

Comment:
Why do we have this here?

Reply:
Looks like the order changes, the cancelEditCallBack was sitting below captionsCallFeatureState, now it's positioned above